contracts/token/ERC721/extensions/ERC721Consecutive.sol

```diff
@@ -87,24 +89,27 @@ abstract contract ERC721Consecutive is IERC2309, ERC721 {
         // minting a batch of size 0 is a no-op
         if (batchSize > 0) {
             require(!Address.isContract(address(this)), "ERC721Consecutive: batch minting restricted to constructor");
-            require(to != address(0), "ERC721Consecutive: mint to the zero address");
             require(batchSize <= _maxBatchSize(), "ERC721Consecutive: batch too large");
```

Comment:
We want to use this for an upcoming contract, hence the speed :) 

Is the batch size necessary since we are essentially minting the offset to the zero address?

Let me know if there is anything else I can do to help test.

Reply:
The max batch size makes sure batches can be processed by offchain tolling. Since these batches to 0 are not actually minted, and don't trigger any event, there is no issue with offchain tolling here.